packages/datatrak-server/src/models/DataTrakSession.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { SessionType, SessionModel } from '@tupaia/server-boilerplate';
+
+export class DataTrakSessionType extends SessionType {
+  public static databaseType = 'datatrak_session';
+}
+
+export class DataTrakSessionModel extends SessionModel {
```

Comment:
I guess we will need to add the hardcoded public permissions that tupaia users have in to here somewhere as well. I'll make a note on the ticket for that one.